### PR TITLE
Updated Windup 2.5.0.Final released version

### PR DESCRIPTION
**Pull request by mareknovotny**
Download Manager link is updated in website for 2.5.0.Final.

@d-s, @jsight, @OndraZizka - when we feel we are ready just merge it, certainly with other changes we would like to include ;)

### CODE REVIEW

**Comment by mareknovotny** [comment]
so hold on for a moment I realized the link needs to be published somehow, I need to wait for a confirmation from responsible person

File: assets/js/windup.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 $(".windup-download-now").click(function() {
-    window.location = 'https://www.jboss.org/download-manager/file/jboss-migrationtoolkit-windup-2.4.0.GA.zip'
+    window.location = 'https://www.jboss.org/download-manager/file/windup-distribution-2.5.0.Final-offline.zip'
```
Comment (d-s) [inline assets/js/windup.js]: Rodney mentioned we should have this file named "jboss-migrationtoolkit-windup-2.5.0.GA.zip" on the Download Manager, same as for 2.4. It's the same file, just renamed. I posted a comment on the JIRA to ask for that update.

**Comment by mareknovotny** [comment]
So everything is ready, download-manager link setup and working.

File: assets/js/windup.js (hunk 1 above)
Comment (OndraZizka) [inline assets/js/windup.js]: The link doesn't work for me:
https://www.jboss.org/download-manager/file/windup-distribution-2.5.0.Final-offline.zip

Comment (OndraZizka) [inline assets/js/windup.js]: I see, it was renamed. Marek is fixing

**Pull request merged**